utility.py

```diff
@@ -0,0 +1,52 @@
+import json
+import logging
+import os
+from parser.apartment import Apartment
+
+
+APARTMENT_FILE_PATH = os.path.relpath("apartment_data.json")
+
+
+def clear_list_of_apartments_before_execute():
+    with open(APARTMENT_FILE_PATH, mode="w") as afp:
+        afp.write("[]")
+
+
+def save_list_of_apartments(apartments: list) -> None:
+    """Save an apartment list into a JSON file by appending to an existing list."""
+
+    if os.path.exists(APARTMENT_FILE_PATH):
+        with open(APARTMENT_FILE_PATH, mode="r") as afp:
+            try:
+                existing_data = json.load(afp)
+            except json.JSONDecodeError:
+                existing_data = []
+    else:
+        existing_data = []
+
+    existing_data.extend(apartments)
+
+    with open(APARTMENT_FILE_PATH, mode="w") as afp:
+        json.dump(existing_data, afp, indent=4)
+
+
+def get_all_apartments() -> list[Apartment]:
+    with open(APARTMENT_FILE_PATH, encoding="utf-8") as afp:
+        return json.load(afp)
+
+
+def suppress(exception: Exception, error_msg: str):
```

Comment:
exception argument can be a tuple and you will be able to except multiple errors at the same time

Reply:
done